tools/import/import.css

```
@@ -108,10 +108,26 @@
  padding-top: var(--spacing-s);
 }
 
-.import div.hidden {
+.import .hidden {
   display: none;
 }
 
+.import button.negative,
+.import button.negative:focus {
+  background-color: var(--gray-100);
```

I think this is my final suggestion. Make the background white.

```suggestion
  background-color: var(--color-background);
```